deploy/common/k8sbased.sh

```diff
@@ -633,6 +633,18 @@ function launch_sensor {
     	extra_helm_config+=(--set "admissionControl.listenOnEvents=${bool_val}")
     fi
 
+    if [[ "${ROX_COLLECTOR_RUNTIME_CONFIG:-false}" == "true" ]]; then
+        extra_helm_config+=(--set "collector.runtimeConfig.enable=${ROX_COLLECTOR_RUNTIME_CONFIG}")
```

Comment:
There are in fact two occurrences of a field `enable` in our charts (coming from scanner V4), but the vast majority uses `enabled`, which is probably what we should be using for new switches of this kind.